### DISCUSSION TITLE
Couple questions!

### DISCUSSION THREAD

**Discussion by maylorian** [roflcoopter/viseron#137] [General]
Hey there!

Many thanks for the project looks interesting a would definately check it out. Just have a couple of questions:

1) Can visceron be configured to do inference on a motion trigger? So when motion is detected, try and identify objects rather that doing so on a live feed.

2) Trying to get some low powered gear for this. In your experience, would a jetson nano be a better fit than a raspberry pi 4 with google coral? (Note sure the coral can run any of the YOLO models) I am gonna be running 4 x 1080p+ cameras.

Thanks again for your hard work!

**Comment by roflcoopter** [comment]
Thanks for showing interest in Viseron!

> 1. Can visceron be configured to do inference on a motion trigger? So when motion is detected, try and identify objects rather that doing so on a live feed.

Yes, this is the default behaviour. Viseron will only start the object detector when motion is detected. You can change this tho if you want to always scan for motion using ```trigger_detector``` config option

> 2. Trying to get some low powered gear for this. In your experience, would a jetson nano be a better fit than a raspberry pi 4 with google coral? (Note sure the coral can run any of the YOLO models) I am gonna be running 4 x 1080p+ cameras.

I have not used either sadly, and the Jetson Nano is currently not supported. However adding support will be very easy since i have a cross-build setup going on in the latest betas. However i would need someone who can test it for me.
An RPi4 with a coral will work very well, but in my experience the YOLOv4 object detection is **much** more accurate so a Jetson Nano is probably the best option.

Do you have any of the hardware today? If so i would recommend you test it on that first and see how you like it before purchasing anything new

**Reply by maylorian** [reply]
Many thanks for your reply!!

I am getting one of the two (and possibly both) anyway. Tinkers gotta tinker :)

Once the nano arrives, I would be more than happy to test this for you!

**Reply by roflcoopter** [reply]
Great! There is already an opened feature request for the Jetson nano, #108, i suggest you subscribe to that issue to follow along with any updates on the matter
